lib/stove-testing-e2e/src/main/kotlin/com/trendyol/stove/testing/e2e/http/HttpSystem.kt

```diff
@@ -86,6 +86,19 @@ interface HttpSystem : PluggedSystem {
         expect: suspend (StoveHttpResponse) -> Unit,
     ): HttpSystem
 
+    /**
+     * Sends a DELETE request to the relative [uri] and expects [StoveHttpResponse]
+     * Use this method when only the status is the important for your case
+     * Provide a [token] in case of Unauthorized with `token = Some("YOUR_TOKEN")`
+     *
+     * Also: [Companion.postAndExpectBodilessResponse]
```

Comment:
`Also: [Companion.postAndExpectBodilessResponse]`  this line is irrelevant. But maybe it is good to create an extension for the tokenless version. What do you think?